qDict/src/main/res/layout/about.xml

```diff
@@ -4,56 +4,73 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 android:background="@color/mmt_white"
-                android:paddingBottom="15dp">
+                android:paddingBottom="15dp"
+                xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/icon_name_constraint_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
 
     <ImageView
```

Comment:
Since we're nesting it one more level, this should be indented one level more. You can see if the automatic formatting of Android Studio or whichever IDE you're using does the trick.

Reply:
OK, I will add some indentations to it with Android Studio. Thanks again for your patience! :)